airgun/entities/subscription.py

```diff
@@ -139,6 +139,19 @@ def read_subscription_manifest_expiration_date_only(self):
             manage_view.close_button.click()
         return manifest_expiration_date
 
+    def is_subscription_manifest_header_message_display(self):
+        """Is header & message present in manage manifest modal box or not"""
+        result = False
+        view = self.navigate_to(self, 'Manage Manifest')
+        view.wait_animation_end()
+        if view.manifest.alert_message.is_displayed:
+            result = True
```

Comment:
```suggestion
        result = view.manifest.alert_message.is_displayed:
```
Can't you just assign it like this? (also removing result = False on line 144) `is_displayed` returns bool

Reply:
Suggested changes should work, will test it and update PR.